src/text-fragment-utils.js

```diff
@@ -869,6 +869,7 @@ export const setDefaultTextFragmentsStyle = ({backgroundColor, color}) => {
         'beforeend', `<style type="text/css">${defaultStyle}</style>`);
   }
   else {
+    applyTargetTextStyle();
```

Comment:
I think if we do this here then we don't need to do it in `text-fragments.js`

Also `applyTargetTextStyle` probably doesn't need to be exported anymore